vg_chart_generator/README.md

```diff
@@ -1,3 +1,12 @@
+# Command Line Video Game Chart Generator
+
+## How to install
+1) Build the distribution:
+$ python setup.py sdist
+
+2) Pip install from generated package:
+pip install dist/vg_chart_generator-1.0.0.tar.gz
+
```

Comment:
Alright so I got it installed now that I've read this. However, I need clarification. If I didn't have this already on my computer, would I have to clone from git, then build the distribution, followed by pip install? Assuming this is the case and not having access to the program on github, how would I install it if it were publicly available? How owuld I be able to build the distribution?